blackberggroup/VA-CCPI
# feat: updated hero image

## What type of PR is this? (check all applicable)


Update Hero image from figma

## Related Tickets & Documents
- Monday Task: https://blackberg-group.monday.com/boards/1931719392/pulses/7370392336

## QA Instructions, Screenshots, Recordings

Check new image in hero from figma: https://www.figma.com/design/YwFbJzXRBZeNr2OGWwSUEU/CCPI-Website?node-id=494-1146&node-type=FRAME&t=9HmGL5T3TQoVdeok-0

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

## Changed Files in this Pull Request ##
- _site/assets/img/home/hero-img.svg
 - assets/img/home/hero-img.svg
